README.md

```diff
@@ -1,4 +1,5 @@
 # File Checker
-[![Build Status](https://travis-ci.org/iattempt/FileChecker.svg?branch=master)](https://travis-ci.org/iattempt/FileChecker)
+[![Build Status](https://travis-ci.org/iattempt/FileChecker.svg?branch=master)](https://travis-ci.org/iattempt/FileChecker) [![codecov](https://codecov.io/gh/iattempt/FileChecker/branch/dev/graph/badge.svg)](https://codecov.io/gh/iattempt/FileChecker)
```

Comment:
@iattempt The build status is shown for the master branch, but the coverage status for the dev branch. Which should I keep?

Reply:
@arnavb please change to master branch.